search_engine_image_thumbnail/models/se_image_field_thumbnail_size.py

```diff
@@ -93,26 +92,22 @@ def _is_model_valid_for_thumbnail(self, model_name):
         return False
 
     @api.depends("model_id")
-    def _compute_field_id_domain(self):
+    def _compute_allowed_field_ids(self):
         for record in self:
-            if not record.model_id:
-                # Eg: NewId
-                record.field_id_domain = FALSE_DOMAIN
-                continue
             domain_fields = []
             if record.model_id:
                 model = self.env[record.model_id.model]
                 domain_fields = model._fields.values()
+            
             names = []
             for field in domain_fields:
                 if self._is_field_valid_for_thumbnail(field):
                     names.append(field.name)
-            record.field_id_domain = json.dumps(
-                [("name", "in", names), ("model_id", "=", record.model_id.id)]
-                if names
-                else FALSE_DOMAIN
-            )
-
```

Comment:
@kobros-tech The error is here.... if you remove the assign directly the domain to the field_id_domain without using the call to `json.dumps` this should work. This pattern is very common in many mocules in the community and it works like a charm. It's also used into Odoo.

```python
record.field_id_domain = (
    [("name", "in", names), ("model_id", "=", record.model_id.id)]
    if names
    else FALSE_DOMAIN
)

Reply:
Yeah, it works now after removing dumps function.

Thanks @lmignon very much, I will keep both approaches but I will archive mine which is pulled here.

See you on the new pull request.

[https://github.com/OCA/search-engine/pull/200](https://github.com/OCA/search-engine/pull/200)